src/clr-angular/data/datagrid/render/header-renderer.ts

```diff
@@ -68,9 +67,9 @@ export class DatagridHeaderRenderer implements OnDestroy {
 
   public setWidth(width: number) {
     if (this.strictWidth) {
-      if (this.columnResizer.columnResizeBy) {
+      if (this.columnResizerService.resizedBy) {
         this.renderer.setStyle(this.el.nativeElement, 'width', width + 'px');
-        this.columnResizer.columnResizeBy = 0;
+        this.columnResizerService.resizedBy = 0;
```

Comment:
Can you move this to the `startResize()` method of the service? That's where you compute `widthBeforeResize`, so it makes sense to reset both of them at the same time.